nmigen_stdio/test/test_serial.py

```diff
@@ -1,21 +1,24 @@
-import unittest
+# nmigen: UnusedElaboratable=no
+
 from nmigen import *
 from nmigen.lib.fifo import SyncFIFO
+from nmigen.lib.io import pin_layout
 from nmigen.back.pysim import *
+from nmigen.test.utils import *
```

Comment:
This module is not called `_test` because (I think) otherwise test runners would get confused. Nevertheless it is not a part of public API.